vcxagency-base/build.sh

```diff
@@ -2,12 +2,13 @@
 
 SCRIPT_DIR_PATH=$(dirname "$0")
 
-INDYSDK_REPO="https://github.com/hyperledger/indy-sdk"
-INDYSDK_REVISION="1.15.0"
+INDYSDK_REPO="https://github.com/Patrik-Stas/indy-sdk"
```

Comment:
Why do we have to use your fork of the repo to set the default logger?

Reply:
Oh yes, I should have clarify that. The default logger is just a "side improvement". But primary change introduced by this PR is exactly building `pgsql` plugin from my IndySDK fork. In particular this commit https://github.com/Patrik-Stas/indy-sdk/commit/cc15191a75bda5970c966dbb7f88098f008ef037 which adds logs to pgsql plugin, enabling troubleshooting in case of connectivity / permissions issues with regard to pgsql.

Reply:
I see, OK. I would be wary of reliance on forked repo, but we will migrate to vdr-tools anyways in the near future anyways.